docs/indicators/url-extraction.md

```diff
@@ -0,0 +1,104 @@
+---
+id: url-extraction
+title: URL Extraction
+---
+
+
+The Cortex XSOAR URL indicator type is built using regex and a formatting script.
+The following describes the URL indicator components and what output you should expect when extracting indicators of type URL.
+
+## URL Indicator Components
+
+There are two components when extracting URL indicators:
+- Regex
+- Formatting script
+
+### Regex

```

Comment:
regex extraction

Reply:
The name of the component is Regex, the extraction is the output.
It's just the name of the two components, a detailed explanation is provided afterward.